test/napi/lib/workers.js

```diff
@@ -10,6 +10,9 @@ const addon = require("..");
 
 // Receive a message, try that method and return the error message
 if (!isMainThread) {
+  // Attempt to reproduce shutdown race condition bug
+  addon.reject_after(new Error("Oh, no!"), 200).catch(() => {});
```

Comment:
This is technically a nondeterministic test, so it's worth at least leaving some kind of hint in the comments that it could become flaky. Maybe just a tag like `// RACE: Attempt to reproduce shutdown race condition bug`?